### PR TITLE
🚁 change output model to template consuming raw model

### PR DESCRIPTION
**Pull request by OsmondJiang**
content -> conceptual
word_count ❌
schema_type ❌
DepotName/_path/CanonicalUrlPrefix/PdfUrlPrefixTemplate -> output model

### CODE REVIEW

**Comment by OsmondJiang** [comment]
no need keep both `word_count` and `wordCount`, blow is the template logic:

image

cc @superyyrrzz  @yufeih

File: src/docfx/build/page/OutputModel.cs [hunk 1]
```diff
@@ -49,7 +53,16 @@ public class OutputModel : InputMetadata
 
         public string SiteName { get; set; }
 
-        public string Conceptual => Content as string;
+        public string DepotName { get; set; }
+
+        [JsonProperty("_path")]
```
Comment (yufeih) [inline src/docfx/build/page/OutputModel.cs]: `_path` can can be removed

Comment (OsmondJiang) [reply src/docfx/build/page/OutputModel.cs]: _path is used to calculate canonical url in template:

image

Comment (yufeih) [reply src/docfx/build/page/OutputModel.cs]: ok, we then need to update template code after next weeks meeting, because `_versionPath` is also missing.

**Pull request merged**